src/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -92,6 +93,8 @@ internal class AuthenticationRequestParameters
         public IDictionary<string, string> ExtraQueryParameters { get; }
         public string Claims { get; }
 
+        public bool IsBrokerEnabled { get; set; }
```

Comment:
Do you need the set here?

Reply:
I'm wondering why the properties aren't read-only. Do they need to be set outside of construction time?

---
In reply to: [254643798](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/874#discussion_r254643798) [](ancestors = 254643798)